build/BUILD.gn

```diff
@@ -20,4 +20,4 @@ config("version") {
   defines = [
     "BRAVE_CHROMIUM_VERSION=\"$chrome_version_string\"",
   ]
-}
```

Comment:
You can probably just leave out this file change from the commit so its history doesn't change.